packages/broker/src/ConfigWizard.ts

```diff
@@ -168,21 +177,16 @@ Object.keys(PLUGIN_DEFAULT_PORTS).map((pluginName) => {
     })
 })
 
-const revealGeneratedPrivateKeyPrompt = {
-    type: 'confirm',
-    name: 'revealGeneratedPrivateKey',
-    message: 'We strongly recommend backing up your private key. It will be written into the config file, but would you also like to see this sensitive information on screen now?',
-    default: false,
-    when: (answers: inquirer.Answers) => {
-        return answers.generateOrImportEthereumPrivateKey === PRIVATE_KEY_SOURCE_GENERATE
+// prompts = prompts.concat(pluginSelectorPrompt).concat(pluginPrompts).concat(revealGeneratedPrivateKeyPrompt)
+export const getEthereumConfigFromAnswers = (answers: inquirer.Answers, config: any) => {
+    config.ethereumPrivateKey = (answers.importPrivateKey) ? answers.importPrivateKey : Wallet.createRandom().privateKey
+    if (answers.revealGeneratedPrivateKey) {
+        logger.info(`This is your node\'s private key: ${config.ethereumPrivateKey}`)
```

Comment:
It would be great if all `getX` methods just return a value a don't do any side effects. Therefore maybe good to move the `logger.info` call outside of this method, and also remove the manipulation of `config` object. 

To me it makes sense to group all config generation into a single methods (as it was in `getConfigFromAnswers`), and pass `privateKeyAnswers` and `pluginAnswers` to that method.

Reply:
There's the call to the method `getEthereumPrivateKeyFromAnswers` that should allow us to display the generated pkey in "runtime" but it still makes the whole logic seem a bit weird.